tests/integration/tasks/test_azure_sql.py

```diff
@@ -142,3 +143,18 @@ def test_check_column_order_append_not_exists(caplog):
         table="non_existing_table_123", schema="sandbox", if_exists="append", df=df
     )
     assert "table doesn't exists" in caplog.text
+
+
+def test_azure_sql_if_failed(caplog):
```

Comment:
Change name to `test_azure_sql_if_failed_break` and add another test for the second option -> `test_azure_sql_if_failed_skip`